packages/mongo/oplog_v2_converter.js

```diff
@@ -0,0 +1,113 @@
+// we are mapping the new oplog format on mongo 5
```

Comment:
It would be nice to post this code [here](https://stackoverflow.com/questions/68936772/mongo-oplog-format-in-5-0-2-contains-s-prefix-on-updates).

And also share with the authors of RedHat Debezium